src/parser/jobs/whm/modules/Lilies.tsx

```diff
@@ -0,0 +1,167 @@
+import {Plural, Trans} from '@lingui/react'
+import Color from 'color'
+import {DataLink} from 'components/ui/DbLink'
+import {Event} from 'event'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {CounterGauge, Gauge as CoreGauge, TimerGauge} from 'parser/core/modules/Gauge'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const LILY_INITIAL_STACKS = 0
+const LILY_MAX_STACKS = 3
+const LILY_60_TIME_REQUIRED = 30000
+const LILY_TIME_REQUIRED = 20000
```

Comment:
Prefer to be explicit, and slightly more readable:
```suggestion
const LILY_INTERVAL_600 = 30000
const LILY_INTERVAL_610 = 20000
```